intro.tex

```diff
@@ -1,54 +1,67 @@
+The Operations team will then carry out an Operations Readiness Review (ORR) to effectively conduct a “full dress rehearsal” of science operations and demonstrate the readiness of the Operations team to execute the 10-year survey. 
+Data collected during commissioning is reprocessed to produce DP2, which will be released 6 months follwoing the completion of the SV Surveys, see S~\ref{tab:dr-one-products}.
+
+\item \textbf{Scenario B}:
+On-sky time in commisisoning is reduced further resulting in the science validation surveys not being completed prior to the construction Operations Readiness Review (ORR). 
+In this scenario, the Operations team would spend up to 3 months prior to commencing the 10-year survey completing any remaining Science Validation Survey observations and releasing DP2 with the exact same data products as planned but 
+
 \end{itemize}
 
-All options will include alert generation of some type, with the major distinction being the relative availability of templates in time, sky position, and filter.
-The principle aspect of this strategy both  in SIT-COM and year 1 operations is incremental template generation.
-In full survey operations, template images for difference image analysis and alert generation are constructed as part of the annual DRP.
-In order to support alert generation in year 1, Rubin will incrementally generate templates during SIT-COM and year 1 using the best images available and covering as much sky  in as many filters as possible.
-Details of the current strategy for alert generation  with incremental templates are given in  \S~\ref{sec:pp}.
+An key point to note is that the contents of DP2 will be unchanged irrespective of which scenario materializes.  
+Only the timing of the release of DP2 and the start of the 10-year survey are difrerent between the two scenarios. 
+In both scenarios it is assumed that the Rubin Construction project delivers an integrated system that can capture, transfer and process science-grade data at the time Operations begins.
+Both scenarios will include alert generation of some type, with the major distinction being the relative availability of templates in time, sky position, and filter.
+
+
 
-Rubin pre-operations is working with the construction project to provide a series of Data Previews based on the data acquired during the SITCOM \svs, as well as any additional science-quality data taken throughout the full commissioning period \citep{RDO-011}.
 
-These options are current at the time of writing, but as this document is "living",  we expect the plans to mature as we approach full survey operations and the extant SIT-COM program emerges and is executed.
+These scenarios presented are current at the time of writing, but as this document is "living", we expect the plans to mature as we approach full survey operations and the extant commissioning program emerges and is executed.
```

Comment:
```suggestion
These two scenarios presented are current as of October 2022, but since this document is ``living,'' we expect the plans to mature as we approach full survey operations and the commissioning program emerges and is executed.
```